includes/promoted-jobs/class-wp-job-manager-promoted-jobs.php

```diff
@@ -149,18 +149,17 @@ public static function is_promoted( $post_id ) {
 	/**
 	 * Update promotion.
 	 *
-	 * @param int  $post_id
-	 * @param bool $promoted
+	 * @param int         $post_id
+	 * @param bool|string $promoted `true` to promoted, `false` to not promoted, `force_delete` to delete.
+	 *                              The deletion is used to force a removal from the feed, deactivating the promotion while syncing.
```

Comment:
I think we should probably update this PHPDoc, right?

Reply:
Yep! Good catch! I updated the previous commit just changing this. ;)
https://github.com/Automattic/WP-Job-Manager/pull/2562/commits/51ebc4b1032916694a0271080d4d8afcf95b8335